### ISSUE TITLE
Missing `testing.B`

### ISSUE THREAD

**Issue by holiman** [AdamKorcz/go-118-fuzz-build#16]
If the file instrumented by the fuzzer, `contracts_test.go` contains both `testing.F` and testing.T` refernces, things work nicely, but if there happen to reside benchmarks in there too, `testing.B`, then compilation fails: 
```
./contracts_test.go_fuzz.go:130:77: undefined: testing.B
```

Should be a trivial thing to add a panicky `B` implementation along with the others, I guess?

**Comment by AdamKorcz** [comment]
> Should be a trivial thing to add a panicky `B` implementation along with the others, I guess?

Yep, should be fine. If you have time to send a PR now, I can get it merged immediately.
